test_fms/mpp/test_mpp_root_pe.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+#***********************************************************************
+#*                   GNU Lesser General Public License
+#*
+#* This file is part of the GFDL Flexible Modeling System (FMS).
+#*
+#* FMS is free software: you can redistribute it and/or modify it under
+#* the terms of the GNU Lesser General Public License as published by
+#* the Free Software Foundation, either version 3 of the License, or (at
+#* your option) any later version.
+#*
+#* FMS is distributed in the hope that it will be useful, but WITHOUT
+#* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+#* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+#* for more details.
+#*
+#* You should have received a copy of the GNU Lesser General Public
+#* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+#***********************************************************************
+
+# This is part of the GFDL FMS package. This is a shell script to
+# execute tests in the test_fms/mpp directory.
+
+# Ed Hartnett 11/29/19
+
+# Set common test settings.
+. ../test_common.sh
+
+
+# Run the test with one processor
+run_test test_mpp_root_pe 1
+
+# Run the test with all processors
+if [ $(command -v nproc) ] ; then
+  nProc=$(nproc)
+  nProc=$( echo $nProc/2 | bc )
+  run_test test_mpp_npes ${nproc}
```

Comment:
needs to be test_mpp_root_pe

Reply:
@colingladueNOAA,  I changed the doxygen section to follow the guidelines in CODE_STYLE document.  I also added an additional run_test line in test_mpp_root_pe.sh to run with nproc number of processors, where nproc is a user-defined variable set to 4 as default.